### PR TITLE
fix: Optimize old events api endpoint

### PR DESCRIPTION
**Pull request by timgl**
## Problem

Noticed [one team in particular](http://metabase-prod-us/question/456-iops-by-team) doing a bunch of iops. Turns out that's the fivetran connector hitting the api/events endpoint in an inefficient way, querying almost 300GB queries for each  call.
image

## Changes

Do the 1-day optimization even if &after is already set.

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/models/event/query_event_list.py [hunk 1]
```diff
@@ -124,7 +131,7 @@ def query_events_list(
                 **hogql_context.values,
             },
             query_type="events_list",
-            workload=workload,
+            workload=Workload.OFFLINE,
```
Comment (fuziontech) [inline posthog/models/event/query_event_list.py:134]: 👍

File: posthog/models/event/query_event_list.py [hunk 2]
```diff
@@ -138,6 +145,6 @@ def query_events_list(
...
```
Comment (fuziontech) [inline posthog/models/event/query_event_list.py:148]: 👍

**Review by mariusandra — APPROVED** [review]
Some legit looking test failures in `posthog/api/test/test_event.py`, though approach looks good and here's a ✅

**Review by fuziontech — APPROVED** [review]
🚢 it!

**Pull request merged**